openedx/core/djangoapps/programs/tests/test_tasks.py

```diff
@@ -311,7 +302,7 @@ def test_failure_to_create_api_client_retries(
                 tasks.award_program_certificates.delay(self.student.username).get()
 
         assert mock_exception.called
-        assert mock_get_api_client.call_count == (tasks.MAX_RETRIES + 1)
```

Comment:
celery and our homegrown had an off by one difference in how many retries occurred. I thought about setting `MAX_RETRIES` to 11 to match the old behavior but ultimately I think the difference is pretty subtle and "10" is a more normal number.